merkle/compact/tree.go

```diff
@@ -168,33 +168,27 @@ func (t *Tree) recalculateRoot(visit VisitFn) error {
 	return nil
 }
 
-// AddLeaf calculates the Merkle leaf hash of the given leaf data and appends it
-// to the tree.
+// AddLeaf calculates the Merkle leaf hash of the given leaf data and appends
+// it to the tree. Returns Merkle hash of the new leaf.
```

Comment:
Put a 'the' before the 'Merkle hash' to make the English flow a little better (like the rest of the comment)?